### PR TITLE
activities and emotions are merged in one table

### LINKED ISSUE #11: Optimize database by merging `activities` and `emotions`
It seems to be a good idea to merge `activities` and `emotions` objects because they are identical. We can just create tables like `activities_and_emotions`, `types_of_activities_and_emotions` instead of having twice more tables. The schemes of the combined tables should be the same as the current schemes but with one additional `enum` field, specifying whether a row is an activity or an emotion.

It should happen for both database and back-end levels (in the back-end the change should simplify the code and probably reduce duplication), I'm not sure whether it should be changed in the API and the part of front-end invisible to the end-user, and I guess activities and emotions should stay separate on the UI level.

### PR DESCRIPTION
**Pull request by VladimirKoryakin**
Relates to #11

### CODE REVIEW

File: database_scheme.sql [hunk 1]
```diff
@@ -1,3 +1,5 @@
+CREATE TYPE class AS ENUM ('activity', 'emotion');
```
Comment (kolayne) [inline database_scheme.sql]: I don't like the name of the type... It should be a bit more descriptive to my mind

**Review by kolayne — COMMENTED** [review]
Hi!
1. There is an issue I've mentioned in a comment to the code line
2. Did you remove the database_scheme.png file on purpose or it's a mistake? Did you forget to update it?
3. (for further reviewers) Please, be careful and note, that #7 is incompatible with this PR, so if #7 will be merged into `mater` before it, it shouldn't be merged directly to master but should be merged into a special branch, where back-end will be updated not to break the version in `master`. In case this PR is merged first, #7 will need to be updated

File: database_scheme.sql (hunk 1 above)
Comment (VladimirKoryakin) [reply database_scheme.sql]: Yes, I thought about it, too. But if we'll make it more informative it'll be much longer. What about naming this field `activity_or_emotion`?

**Comment by VladimirKoryakin** [comment]
And what about the database_scheme.png I think that we have quite simple database scheme now so the picture of it might be needless. Because of these reasons I decided not to update the database_scheme.png and just delete this file from the repository.

File: database_scheme.sql (hunk 1 above)
Comment (kolayne) [reply database_scheme.sql]: I'm not sure if it's perfect but much better

**Review by kolayne — APPROVED** [review]

**Review by tanya-kta — APPROVED** [review]

**Comment by kolayne** [comment]
I think this should be merged into an intermediate branch, where the back-end will be updated to work with the new scheme

**Comment by kolayne** [comment]
@VladimirKoryakin, even though #7 is merged, so this PR is not compatible with master anymore, please don't start working on combining them, because we're probably going to make more changes to the schema/architecture, which will relate to activities/emotions and their types.

I'm closing this PR to prevent it from being merged accidentally but I believe the work you've done will be used soon, so don't delete the branch!

**Comment by kolayne** [comment]
Hi! I'm going to fix merge conflicts in the `merge_activities_and_emotions` branch, open a PR and request your review on it. Stay tuned!

**Pull request closed without merging**